scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -468,58 +515,68 @@ We can correct this by applying a "shrinkage" procedure, which will adjust large
 
 To do this, we will use the `lfcShrink()` function, but first we need to know the name and/or position of the "coefficient" that was calculated by `DESeq()`, which we can do with the `resultsNames()` function.
 
-```{r}
+```{r coefficient}
 # identify position of coefficient
 DESeq2::resultsNames(deseq_object)
 ```
 
 
-```{r}
-# appyly logFC shrinkage
+```{r shrinkage}
+# appyly logFC shrinkage using the default model
 shrink_results <- DESeq2::lfcShrink(deseq_object, res = deseq_results, coef = 2, type = "apeglm")
+head(shrink_results)
 ```
 
-Now we can get the updated results and combine with the `rowData` from the SCE object to extract the gene symbol. 
+If you look at our `shrink_results` object, we see that the genes are labeled with the ensembl gene identifiers, as those were the rownames of the pseudo-bulked SCE we used as input to build our `DESeq2` object.
+Although some of us may have all of the identifiers memorized by heart, it can be useful to have a human readable symbol in our results.
+Before we save the results as a file, we will grab the gene symbols from the `rowData` of our original SCE object and add them as a new column.
 
-```{r}
-# create a data frame with the results 
-deseq_results <- shrink_results %>%
+```{r add gene symbol}
+# convert shrunken results to a data.frame and add ensembl id as a column
```

Comment:
I don't know if we want to get into the difference between a tibble and a data.frame. But 
```suggestion
# convert shrunken results to a tibble and add Ensembl id as a column
```